### PR TITLE
Docker-compose and justfile

### PR DESCRIPTION
**Pull request by AetherUnbound**
This PR prepares the project for a move from glitch onto our own infrastructure by standardizing its deployment using `just` and `docker-compose`. Fortunately this is a pretty simple app, so nothing too complex (unless we have to deal with static files, in which case I'll `[REDACTED]`). I've also upgraded to Python 3.9, because Glitch was the one holding us back and I want to use the walrus operator for stuff.

### CODE REVIEW

**Comment by mpuckett159** [comment]
Can we update the docs to include the just commands? I spent a bit trying to figure out why they didn't work anymore lol

**Comment by AetherUnbound** [comment]
Ah, great call - will do!

**Review by mpuckett159 — APPROVED** [review]
Excellent, looks great, glad we're moving to just.

**Comment by AetherUnbound** [comment]
I'm actually going to cut the OpenOversight link changes off into a separate PR. We won't want to deploy that until we launch OO.

File: justfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+IS_PROD := env_var_or_default("IS_PROD", "")
+COMPOSE_FILE := "-f docker-compose.yml " + if IS_PROD == "true" {""} else {"-f docker-compose.override.yml "}
+DC := "docker-compose " + COMPOSE_FILE
+RUN := DC + " run --rm app"
+set dotenv-load := false
```
Comment (sarayourfriend) [inline justfile:5]: I feel like at this point `just` should just have a `dc` variant with all this built in 😂

File: README.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ Licenses are looked up in the dataset and if a vehicle is found, that data is di
...
 ## Development
 
-### If you just want to run the app
+We use the [`just` command runner](https://github.com/casey/just), you'll need to install it before running any steps below.
 
-Build the image with `docker build -t <tag> .`. Run it with `docker run -p 5000:5000 <tag>`.
+### Build and run the app
+
+Build the image with `just build` and run it using `just up`.
+The server will be available at port `3030`.
```
Comment (sarayourfriend) [inline README.md:17]: Should we add a small note about how to use `IS_PROD`? Is it just `IS_PROD=true just up`?

**Review by sarayourfriend — APPROVED** [review]

File: justfile (hunk 1 above)
Comment (AetherUnbound) [reply justfile:5]: Seriously! I wish we could host a common `justfile` somewhere and import it here so we didn't have to keep repeating ourselves. I might end up doing a repo-sync similar to some other projects where the `justfile` is defined in one place and updated everywhere. I hate having to keep updating the dozen or so projects I work with!

File: README.md (hunk 2 above)
Comment (AetherUnbound) [reply README.md:17]: Great idea!

File: justfile (hunk 1 above)
Comment (AetherUnbound) [reply justfile:5]: Drat lol: https://github.com/casey/just/issues/906

**Pull request merged**